include/api.php

```diff
@@ -3880,6 +3933,72 @@ function api_friendica_direct_messages_search($type){
 	api_register_func('api/friendica/direct_messages_search', 'api_friendica_direct_messages_search', true);
 
 
+	/**
+	 * @brief return data of all the profiles a user has to the client
+	 *
+	 * @param string $profile_id optional parameter to provide the id of the profile to be returned
```

Comment:
The function does have a single parameter - but there are two described in the doxygen header. Could you correct it?